distribution/std-lib/Standard/src/Base/Data/Vector.enso

```diff
@@ -67,26 +76,44 @@ new_builder = Builder.new
 ## Converts a polyglot value representing an array into a vector.
 
    Arguments:
-   - arr: The array value to wrap into a vector.
+   - arr: The array value to wrap into a vector. This value must conform to the
+     Truffle polyglot notion of 'array'.
```

Comment:
Sounds like implementation details. Do we expect the user to know what a _Truffle_ is?